### PR TITLE
databases-week3

### PR DESCRIPTION
**Pull request by hparul**

### CODE REVIEW

File: databases/week3/task1.sql [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+--Get only specific number of meals fx return only 5 meals
+SELECT * FROM meal LIMIT 4;
+--Get the meals that have good reviews
+SELECT meal.title,meal.id,Review.stars,Review.description from meal
+join Review on meal.id=Review.meal_id
+where stars>=4; 
+--Get reservations for a specific meal sorted by created_date 
+select Reservation.number_of_guests,meal.title,Reservation.created_date from meal
+join Reservation on meal.id=Reservation.meal_id
+where meal.id=2
```
Comment (Elia-Git) [inline databases/week3/task1.sql:170]: Why only **meal.id=2?** ignore this line
hint ```
 SELECT meal.Title, reservation.meal_Id, SUM(reservation.Number_of_guests), reservation.Created_date AS Res_created_date
FROM reservation
   JOIN meal ON (meal.id=reservation.meal_id)
GROUP BY reservation.created_date,meal.Title
ORDER BY reservation.created_date;
```

**Review by Elia-Git — COMMENTED** [review]
Good job. Looking forward to see your implemntations.

File: databases/week3/task1.sql [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+
+/*  Get meals that has a price smaller than a specific price fx 90
+Get meals that still has available reservations
+Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+Get meals that has been created between two dates
+Get only specific number of meals fx return only 5 meals
+Get the meals that have good reviews
+Get reservations for a specific meal sorted by created_date
+Sort all meals by average number of stars in the reviews */
```
[the remark this answers is not in the document]
Comment (hparul) [reply databases/week3/task1.sql]: done

File: databases/week3/task1.sql [hunk 3]
```diff
@@ -0,0 +1,181 @@
+use hyf_lesson3;
+
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (hparul) [reply databases/week3/task1.sql]: done

**Pull request merged**